software/applications/HelloIOIO/build.gradle

```diff
@@ -6,11 +6,26 @@ android {
         minSdkVersion 14
         targetSdkVersion 30
     }
+
+    flavorDimensions "integration"
+    productFlavors {
+        code {
+            dimension "integration"
+        }
+        aar {
+            dimension "integration"
+        }
+    }
+
 }
 
 dependencies {
-    implementation 'com.github.ytai.ioio:IOIOLibAndroid:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidBluetooth:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidAccessory:5.07'
-    implementation 'com.github.ytai.ioio:IOIOLibAndroidDevice:5.07'
+    codeImplementation project(':IOIOLibAndroidBluetooth')
```

Comment:
What is this call? I'm not familiar with this and don't see it in the gradle [docs](https://docs.gradle.org/current/userguide/java_library_plugin.html)